src/plugins/irc/irc-protocol.c

```diff
@@ -4506,15 +4520,18 @@ IRC_PROTOCOL_CALLBACK(366)
                     if (strcmp (weechat_infolist_string (infolist, "type"),
                                 "nick") == 0)
                     {
-                        ptr_option = weechat_config_get (weechat_infolist_string (infolist,
-                                                                                  "prefix_color"));
-                        length +=
-                            ((ptr_option) ? strlen (weechat_color (weechat_config_string (ptr_option))) : 0) +
```

Comment:
Couldn't really figure out what this line is trying to accomplish because it differs from all the other color lengths. What difference does the option make to the length of the color returned by `weechat_color`, isn't it always the same anyway? Also, not being an option meant that no characters were allocated for prefix color which seemed weird.

If this line actually was required for something, I can leave it in as it was.